src/pages/pupils/beta/programmes/[programmeSlug]/units/[unitSlug]/lessons.tsx

```diff
@@ -5,14 +5,20 @@ import curriculumApi2023 from "@/node-lib/curriculum-api-2023";
 import { LessonListingBrowseData } from "@/node-lib/curriculum-api-2023/queries/pupilLessonListing/pupilLessonListing.schema";
 import getPageProps from "@/node-lib/getPageProps";
 import { resolveOakHref } from "@/common-lib/urls";
-import { URLParams } from "@/pages/teachers/programmes/[programmeSlug]/units/[unitSlug]/lessons";
+import { getStaticPaths as getStaticPathsTemplate } from "@/pages-helpers/get-static-paths";
 
-export { getStaticPaths } from "@/pages/teachers/programmes/[programmeSlug]/units/[unitSlug]/lessons";
```

Comment:
This was a pretty dangerous anti-pattern which was written in response to sonarCloud's complaints about code duplication on pages.